docs/modules/ROOT/pages/index.adoc

```diff
@@ -228,24 +228,25 @@ This specification defines a _<<C2PA assertion>>_ known as an *<<_identity_asser
 
-IMPORTANT: A single signature from the _<<_credential_subject,credential subject>>_ covers the _list_ of assertions. An implementation MUST NOT present each assertion separately for signature.
+IMPORTANT: A single signature from the _<<_credential_subject,credential subject>>_ covers the entire `tbs` data structure, including the _list_ of assertions. An implementation MUST NOT present each assertion separately for signature.
 
-For each assertion listed in the `referenced_assertions` field of the *<<_identity_assertion,identity assertion>>,* an assertion with the same `url`, `alg`, and `hash` values MUST also be listed in the `assertions` field of the _<<C2PA claim>>_ in which the *<<_identity_assertion,identity assertion>>* appears.
+For each assertion listed in the `tbs.referenced_assertions` field of the *<<_identity_assertion,identity assertion>>,* an assertion with the same `url`, `alg`, and `hash` values MUST also be listed in the `assertions` field of the _<<C2PA claim>>_ in which the *<<_identity_assertion,identity assertion>>* appears.
 
-NOTE: The list of _<<_referenced_assertions,referenced assertions>>_ in the `referenced_assertions` field of the *<<_identity_assertion,identity assertion>>* is necessarily a subset of the assertions listed in the `assertions` field of the _<<C2PA claim>>_ because the *<<_identity_assertion,identity assertion>>* may not refer to itself. The list of assertions may be further subsetted. For example, different _<<_credential_subject,credential subjects>>_ could attest to different assertions in a single _<<C2PA Manifest>>._
+NOTE: The list of _<<_referenced_assertions,referenced assertions>>_ in the `tbs.referenced_assertions` field of the *<<_identity_assertion,identity assertion>>* is necessarily a subset of the assertions listed in the `assertions` field of the _<<C2PA claim>>_ because the *<<_identity_assertion,identity assertion>>* may not refer to itself. The list of assertions may be further subsetted. For example, different _<<_credential_subject,credential subjects>>_ could attest to different assertions in a single _<<C2PA Manifest>>._
 
 Multiple identity assertions may be used in the same _<<C2PA Manifest>>_ to describe the distinct roles of multiple _<<_actor,actors>>_ in creating a single _<<C2PA asset>>._ This is illustrated in the xref:multiple-identity-assertions[multi-author example] from the conceptual overview.
 
-The `referenced_assertions` entry of the *<<_identity_assertion,identity assertion>>* MUST NOT reference any assertion more than once.
+The `tbs.referenced_assertions` entry of the *<<_identity_assertion,identity assertion>>* MUST NOT reference any assertion more than once.
```

Comment:
Is there a validation item to check this (and fail accordingly)?

Reply:
Will check

Reply:
There wasn't. Since this isn't directly related to the `tbs` addition, I've pulled that out into PR #80.